src/group_gui.cpp

```diff
@@ -573,7 +573,7 @@ class VehicleGroupWindow : public BaseVehicleListWindow {
 				uint32 occupancy = 0;
 				size_t vehicle_count = this->vehicles.size();
 
-				for (uint i = 0; i < vehicle_count; i++) {
+				for (uint32 i = 0; i < vehicle_count; i++) {
```

Comment:
probably switch the whole thing out with a range-for loop, since you're changing it anyway